pkgs/check_mk_agent/default.nix

```diff
@@ -79,14 +87,8 @@ let
 in
 stdenv.mkDerivation rec {
```

Comment:
I think you don't need `rec` here anymore, since you're not referring to `version` in the `fetchFromGitHub` call anymore...

Reply:
thx!